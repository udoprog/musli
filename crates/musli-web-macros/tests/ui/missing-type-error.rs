musli_web_macros::define! {
    impl Broadcast for Broadcast {
        impl Event for String;
    }
}

fn main() {}