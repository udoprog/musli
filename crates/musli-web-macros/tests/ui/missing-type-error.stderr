error: Expected corresponding `type` declaration for `Broadcast`
 --> tests/ui/missing-type-error.rs:2:24
  |
2 |     impl Broadcast for Broadcast {
  |                        ^^^^^^^^^
